### PR TITLE
About Page

### LINKED ISSUE #29: About page
Perhaps we should start thinking of a solid about page? I'm not quite the writer, can explain the app and the motives behind it but need help crafting words.

help? @gr2m @nickcolley @Charlotteis @renrutnnej

### PR DESCRIPTION
**Pull request by gr2m**
This is a base of discussion for #29. 

---

closes #29

### CODE REVIEW

File: www/about.html [hunk 1]
```diff
@@ -25,8 +25,71 @@
 
     <main>
       <a href="/">← Return to the tracker</a>
-      <h2>About</h2>
-      <p>Tracker is a concise and small application intended to help you get started with Hoodie. It's a basic tracker of all sorts. The mechanics let you keep track of anything you want plus its amount.</p>
+      <h2>What is Hoodie Tracker?</h2>
+
+      <p>
+        <img src="/assets/low-profile-dog.png" alt="wiggle waggle Low-Profile Dog" class="low-profile-dog">
```
Comment (nickcolley) [inline www/about.html]: For the alt tag could we go the extra mile and be more descriptive?

'Illustration of Hoodie's Low-Profile Dog Mascot waving'

File: www/about.html [hunk 2]
```diff
@@ -25,8 +25,71 @@
...
+        <a href="https://github.com/hoodiehq/editorial/issues/71">
+          How to get started with the new Hoodie (coming soon)
+        </a>
+      </p>
+
+      <h3>
+        Hoodie Camp Release
+      </h3>
+
+      <p><img src="/assets/hoodie-camp.png" alt="Hoodie Camp" class="hoodie-camp"></p>
```
Comment (nickcolley) [inline www/about.html]: 'Illustration of a campsite'

**Comment by nickcolley** [comment]
Looks great! :+1:

**Comment by gr2m** [comment]
@nickcolley amended the image alt tags

**Comment by jsimplicio** [comment]
LGTM :+1:

**Comment by gr2m** [comment]
when you we have two LGTMs, you can merge the PR :) I can do it if you like, otherwise I’ve put some instructions how it works with a rebase on latest `master` branch here: https://github.com/hoodiehq/hoodie-app-tracker/pull/33#issuecomment-195522626

**Pull request merged**